index.html

```diff
@@ -332,23 +437,193 @@ <h4>SvgRenderingTemplate</h4>
       "name": "Bachelor of Science and Arts"
     }
   },
-  <span class="highlight">"renderMethod": [{
-    "id": "https://example.edu/credentials/BachelorDegree.svg",
-    "type": "SvgRenderingTemplate",
-    "name": "Portrait Mode",
-    "css3MediaQuery": "@media (orientation: portrait)",
-    "digestMultibase": "zQmAPdhyxzznFCwYxAp2dRerWC85Wg6wFl9G270iEu5h6JqW"
-  }]
+  <span class="highlight">"renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    "template": {
+      "id": "https://example.edu/credentials/BachelorDegree.svg",
```

Comment:
Maybe we should drop the file extension from the URL, so people don't rashly assume it has meaning. For example https://github.com/bigbluehat/this-is-not-a.pdf

```suggestion
      "id": "https://example.edu/credential-templates/BachelorDegree",
```